nncf/tensorflow/quantization/algorithm.py

```diff
@@ -428,8 +429,12 @@ def _run_batchnorm_adaptation(self, model: tf.keras.Model) -> None:
         self._bn_adaptation.run(model)
 
     def _get_quantizer_setup(self, model: tf.keras.Model) -> TFQuantizationSetup:
+        # pylint:disable=too-many-statements
```

Comment:
Please refactor the code to fix the pylint issue.

Reply:
done